src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -87,7 +93,10 @@ public HostStatus GetHostStatus()
         {
             HostStatus status = new HostStatus
             {
-                Id = _scriptHostManager.Instance?.ScriptConfig.HostConfig.HostId
+                Id = _scriptHostManager.Instance?.ScriptConfig.HostConfig.HostId,
+                WebHostSettings = _webHostSettings,
```

Comment:
Thinking we should only return WebHostSettings when running locally - not really useful otherwise I think?